modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -3,11 +3,12 @@
 
 == What is Salt Bundle?
 
-The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, additional Python modules, and libraries.
```

Comment:
Not sure if it's a proper rewording as there is only the modules required for salt, what are the modules additional to? We just imlementing the ways to add additional modules inside with `pip`.

Reply:
ok, I reverted this